.ci/common.sh

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 SAP SE or an SAP affiliate company
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For all steps, concourse will set the following environment variables:
+# SOURCE_PATH - path to component repository root directory.
+if [[ -z "${SOURCE_PATH}" ]]; then
+  export SOURCE_PATH="$(readlink -f "$(dirname ${0})/..")"
+else
+  export SOURCE_PATH="$(readlink -f "${SOURCE_PATH}")"
```

Comment:
It is generally recommended to assign and export separately. See https://github.com/koalaman/shellcheck/wiki/SC2155
In this case the change would be:
```bash
if [[ -z "${SOURCE_PATH}" ]]; then
  SOURCE_PATH="$(readlink -f "$(dirname "${0}")/..")"
else
  SOURCE_PATH="$(readlink -f "${SOURCE_PATH}")"
fi
export SOURCE_PATH
```